rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -410,8 +418,8 @@ public void registerBtcTransaction(
                 case PEGOUT_OR_MIGRATION:
                     logger.debug("[registerBtcTransaction] This is a peg-out or migration tx {}", btcTx.getHash());
                     processPegoutOrMigration(btcTx);
-                    if (svpIsOngoing()) {
-                        updateSvpFundTransactionValuesIfPossible(btcTx);
+                    if (svpIsOngoing() && isTheSvpFundTransaction(btcTx)) {
```

Comment:
Should this be moved to a method to abstract these two mandatory conditions to process a fund transaction? It could be `isSvpOngoingAndSvpFundTranscation`, wdyt? 

Btw, regarding this method, `svpIsOngoing`, according to Java Standard naming conventions should have been named `isSvpOngoing` instead.

Reply:
Agree with @nathanieliov

Reply:
methods should do just one thing :)

Reply:
also i think is easier to read to have both conditions separately

Reply:
name updated! Thanks guys